examples/intro_to_observations.jl

```diff
@@ -0,0 +1,145 @@
+# # Intro to observations
+#
+# This example demonstrates the features of `OneDimensionalTimeSeries`
+# when constructed from  "synthetic observations" generated by an Oceananigans `Simulation`.
+#
+# ## Install dependencies
+#
+# First let's make sure we have all required packages installed.
+
+# ```julia
+# using Pkg
+# pkg"add OceanTurbulenceParameterEstimation, Oceananigans, CairoMakie"
+# ```
+
+# First we load few things
+pushfirst!(LOAD_PATH, joinpath(@__DIR__, ".."))
+
```

Comment:
```suggestion

```

Reply:
I thikn we need to add some lines to install packages for this to work right?

Reply:
It depends what "for this to work" means? Under which work flow.

Anyhow, just leave it here for now. We can deal with it just before merging?

Reply:
It looks like we did have those lines! I rmeoved it.